kuma/contributions/jinja2/contributions/contribute.html

```diff
@@ -4,12 +4,6 @@
 
 {% block title %}{{ page_title(_('Contribute')) }}{% endblock %}
 
-{% block site_js %}
-    {{ super() }}
-    <script src="https://checkout.stripe.com/checkout.js"></script>
-    {% javascript 'contribute' %}
-{% endblock %}
```

Comment:
@jwhitlock This is also done at the bottom of the page so, this looks like duplication to me. If not, let me know and I will revert.

Reply:
👍 I think you're right, this is duplicated and should be removed. Thanks!